frontend/packages/core/src/tab.tsx

```diff
@@ -81,12 +81,16 @@ const TabPanel = styled(MuiTabPanel)({
 
 export interface TabsProps extends Pick<MuiTabsProps, "value" | "variant"> {
   children: React.ReactElement<TabProps> | React.ReactElement<TabProps>[];
+  setIndex?: (index: string) => void;
```

Comment:
nit: reading this implies that it's triggering behavior instead of on an action (e.g. `onChange`) and index implies this should be an integer. I know we use the word index internally to this component but it should probably be value.